Mathlib/Data/List/RepeatSelf.lean

```diff
@@ -0,0 +1,24 @@
+/-
+Copyright (c) 2025 Shimanonogov Igor. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Shimanonogov Igor
+-/
+import Mathlib.Data.List.Defs
+
+/-!
+# repeatSelf
+
+Proves various lemmas about `List.repeatSelf`.
```

Comment:
It's good that you're trying to be mindful about keeping the Defs file short: at the same time, having one single file with one lemma (with a five line proof) is unusual. Sometimes, there are good reasons for this (e.g., imports) --- but in this case, the lemma could either be
- right after the definition `repeatSelf`
- in a `Lemma.lean` file